app/forms/declaration/DepartureTransport.scala

```diff
@@ -56,6 +65,18 @@ object TransportCodes {
   val InlandWaterway = "8"
   val Unknown = "9"
 
+  val extractModeOfTransportValue: PartialFunction[String, String] = {
+    case Maritime                    => "Sea transport"
```

Comment:
I don't think we should be converting code -> display value like this.  That's what the messages are for.

Reply:
Done